api/src/main/java/io/grpc/ManagedChannelRegistry.java

```diff
@@ -99,10 +99,10 @@ public int compare(ManagedChannelProvider o1, ManagedChannelProvider o2) {
   public static synchronized ManagedChannelRegistry getDefaultRegistry() {
     if (instance == null) {
       List<ManagedChannelProvider> providerList = ServiceProviders.loadAll(
-          ManagedChannelProvider.class,
```

Comment:
Revert all changes to this file? I only see reformatting.

Reply:
Done.